cli/src/commands/redo.rs

```diff
@@ -28,6 +28,14 @@ use crate::ui::Ui;
 /// Redo the most recently undone operation
 ///
 /// This is the natural counterpart of `jj undo`.
+///
+/// Repeated invocations of `jj undo` and `jj redo` are meant to feel intuitive
+/// if you are used to the model of an "undo stack". However, the behavior is
+/// not completely identical in complicated cases. If interested in the details,
+/// experiment and compare the command output to `jj op log` output and/or see
+/// the comments in the source code.
```

Comment:
I'm not sure who this comment is helping. Is "undo stack" a common term people use? I've never heard it outside our discussion about implementing the feature. In what way does our implementation work differently than what people think of as an "undo stack"?

"just try it out" and "read the source code" don't seem helpful either.

I would consider making a reference to the undo/redo feature of GUI apps / text editors. Our implementation works exactly the same on the surface. That should be intuitive for most users. (Our implementation is more powerful, but only once you dive into `jj operation` commands.)

Reply:
I like it much better now, I think this should be clear and intuitive for newcomers. Thanks!